util/dashboard/gen_dashboard_entry.py

```diff
@@ -102,7 +178,7 @@ def gen_specboard_html(hjson_path, rel_hjson_path, outfile):
     if dashboard_validate.validate(obj) == 0:
         log.info("Generated dashboard object for " + str(hjson_path))
     else:
-        log.fail("hjson file import failed")
+        log.error("hjson file import failed")
```

Comment:
Isn't `fail` fatal?  Shouldn't the documents build fail if the HJSON import fails?

Reply:
It is still uncertain. It has been ongoing discussing whether the tool should create error but proceed as much as we can, or just raise an Exception there, or abruptly exist .

Reply:
Understood.